src/js/routes/main-router.jsx

```diff
@@ -35,6 +37,8 @@ const MainRouter = ({ user }) => {
             )
           }
         />
+        <Route path='/directory' render={DirectoryContainer} />
```

Comment:
`render` receives a function -- `component` can return a component like this, but not `render`.